jenkins/pipeline-parallel-mtr.groovy

```diff
@@ -199,6 +199,80 @@ void checkoutSources() {
     """
 }
 
+def downloadCcache() {
```

Comment:
maybe it is better to add this function as a shared library so it can be reused for other products?